core/src/main/scala/fs2/io/ssh/Process.scala

```diff
@@ -86,16 +81,16 @@ final class Process[F[_]: Concurrent: ContextShift] private[ssh] (
   private[this] def ioosToSink(ioosF: F[IoOutputStream]): Pipe[F, Byte, Unit] = { in =>
     Stream.eval(ioosF) flatMap { ioos =>
       val written = in.chunks evalMap { chunk =>
-        val bytes = chunk.toBytes
-        val buffer = new ByteArrayBuffer(bytes.values, bytes.offset, bytes.length)
+        val bytes = chunk.toArray
```

Comment:
This should also be reverted to the previous implementation, but using `chunk.toArraySlice`. This results in fewer allocations when `Chunk` is already backed by an `Array[Byte]`.

Reply:
Same here - good to document this too

Reply:
👍